Classic Momentum.Rmd

```diff
@@ -0,0 +1,70 @@
+output:
+  html_document: 
+    df_print: paged
+    number_sections: yes
+    toc: yes
+    toc_float: true
+    toc_depth: 3
+    code_folding: show
+editor_options:
+  chunk_output_type: inline
+---
+
+# Learning Goals
+
+1. Understand that momentum is conserved in collisions.
+
+2. Understand the difference between **elastic** and **inelastic** collisions
+
+3. Be able to balence and compute the momentum on objects.
+
+# Background
+
+Momentum is a quality of everything! It is the quantity of motion of a moving body, measured as a product of its mass and velocity. Momentum by itself isn't all that interesting, only when it comes to collisions! Let's talk about two different kinds of collisions first.
+
+**Elastic collision:** An elastic collision is an encounter between two bodies in which the total momentum and kinetic energy of the two bodies remains the same. In an ideal, perfectly elastic collision, there is no net conversion of kinetic energy into other forms such as heat, noise, or potential energy.
+
+**Inelastic Collision:** An inelastic collision, in contrast to an elastic collision, is a collision in which momentum and kinetic energy is not conserved due to the action of internal friction.
+
+Because we don't want to have to worry about pesky things like heat, noise, and potential energy, we are going to work with elastic collisons. Now consider the momentum $\rho$ of an object of mass $m$ and velocity $v$. Momentum is then.
+$$
+\rho = mv
+$$
+Because momentum is **conserved** in elastic collisions, momentum of a system is the summation of is parts. Take for example two cars moving in the same direction with masses $m_{1}$ and $m_{2}$ and velocities $v_{1}$ and $v_{2}$ respectively. The total mass of the system is then:
+
+$$
+\rho_{total}=m_{1}v_{1}+m_{2}v_{2}=\rho_{1}+\rho_{2}
+$$
+Because of this conservation, we can explore the concept of many differing colliding objects seen below in our example.
+
+# Example
+
+Two cars are barreling towards another car in a future ellastic collision. There masses and velocities are listed below. Lets use R to calculate the total mass of the system!
+
+```{r}
+library(tidyverse)
+
+# session info
+masses <- c(300, 400, 600)
+velocties <- c(20, 10, -15)
+
+total_momentum <- sum(masses*velocties)
+print(total_momentum)
+
+```
+
+And there you go! The total momuntum of the system is 1000! More specifically it is $kg⋅m/s$. This implies that though to one car is hitting the other two. The three will still continue onwards in the direction of the original two.... though perhaps slower than before.
+
+# Questions
+
+1. You just got your BB gun for Christmas but all is for naught because an astroid is bearing down on Earth! The astroid is $1  \times 10^{15} kg$ and moving at $2 \times 10^{4} m/s$. Fortunatly you have quite a few BBs which weigh $0.0002 kg$ and you can fire them at $60 m/s$. How many BBs will it take to stop the astroid and save Earth?
```

Comment:
give instructions to the students that help them understand how they are supposed to solve this: 
 - please show all steps
- use R to do this calculation

Reply:
Also maybe let them know as a reminder that this is based on an elastic collision